Lib/sqlite3/__main__.py

```diff
@@ -10,9 +10,9 @@
 from argparse import ArgumentParser
 from code import InteractiveConsole
 from textwrap import dedent
+from _colorize import get_theme, theme_no_color
 
-
-def execute(c, sql, suppress_errors=True):
+def execute(c, sql, suppress_errors=True, theme=theme_no_color):
```

Comment:
Please leave out unrelated style changes.

```suggestion

def execute(c, sql, suppress_errors=True, theme=theme_no_color):
```

Reply:
Apologies, this was accidentally introduced at some point!